File: 13_factorialize.js
// factorialize a number and return the result

function factorialize(num){
    var result = 1;

    for (var i=1; i<=num; i++){
        result *= i;
    }
    return result;
}

// recursive way
function recursiveFact(num){
    if(num > 1){
        return num * recursiveFact(num - 1);
    }
    else{
        return 1;
    }
}

console.log(factorialize(6));
console.log(recursiveFact(6));

File: 13_factorialize.py
# factorialize a number and return the result

def factorialize(num):
    result = 1
    for i in range(1, num+1):
        result *= i
    return result


# recursive way
def recursiveFact(num):
    return num * recursiveFact(num-1) if num >1 else 1

print(factorialize(6))
print(recursiveFact(6))